react/tic-tac-toe/src/redux/game/reducer.js

```diff
@@ -1,4 +1,4 @@
-import { completeReducer, createReducer, completeState } from 'redux-recompose';
+import { completeReducer, createReducer, completeState, onReadValue } from 'redux-recompose';
```

Comment:
Why is this `onReadValue` here?, check the status of your base branch and this branch :)